packages/kbn-es/src/serverless_resources/project_roles/security/roles.yml

```diff
@@ -45,7 +45,7 @@ viewer:
         - feature_siem.read
         - feature_siem.read_alerts
         - feature_siem.endpoint_list_read
-        - feature_securitySolutionCases.read
```

Comment:
@azasypkin would I keep both versions since the original `feature_securitySolutionCases` isn't actually gone, just deprecated?